cinn/ir/ir_schedule.cc

```diff
@@ -1201,6 +1201,58 @@ Expr IRSchedule::CacheWrite(const Expr& block, int write_buffer_index, const std
   return *find_cache_block.begin();
 }
 
+struct InsertExpr : public ir::IRMutator<> {
```

Comment:
Is it possible that InsertExpr can be a tool used somewhere else? If yes, suggest to put it at some IR utility file. Our ir_schedule.cc is getting long now.

Reply:
Ok, I plan to move all IR utility codes to a new file in my next PR.